agent/command/results_xunit_parser_test.go

```diff
@@ -222,17 +224,19 @@ func TestXMLToModelConversion(t *testing.T) {
 			ProjectRef: &model.ProjectRef{},
 			Task:       &task.Task{Id: "TEST", Execution: 5},
 		}
-
+		sender := send.MakeInternalLogger()
+		logger := client.NewSingleChannelLogHarness("", sender)
 		Convey("when converting the results to model struct", func() {
 			tests := []testresult.TestResult{}
 			logs := []*model.TestLog{}
 			for _, testCase := range res[0].TestCases {
-				test, log := testCase.toModelTestResultAndLog(conf)
+				test, log := testCase.toModelTestResultAndLog(conf, logger)
 				if log != nil {
 					logs = append(logs, log)
 				}
 				tests = append(tests, test)
 			}
+			So(logger.Close(), ShouldBeNil)
```

Comment:
Can this test also assert somewhere that the test data added [here](https://github.com/evergreen-ci/evergreen/blob/a37cefa7b9b54209c2fc95c75116ea06db474ba8/agent/command/testdata/xunit/junit_3.xml#L7-L8) translates into sensible test start and end times?

Reply:
Ah yes, thanks